### ISSUE TITLE
Can't find app win

### ISSUE THREAD

**Issue by Bite-A-Rock** [Athou/moonlight-openelec-raspberrypi2#2]
Hello,
  Very cool idea. Would love to get this working. Installed as instructed. Running start.sh simply states "Can't find app win"  There is no reference to win in the README nor is thre a file called 'win', not sure what the script is looking for.

Thanks!

**Comment by Athou** [comment]
Good point, i'll update the readme. Actually you have to replace "win" with the name of a game in your nvidia gamestream preferences. I've added mstsc.exe in gamestream and called it "win" so that I can stream the whole screen and not just a specific game.

**Comment by Bite-A-Rock** [comment]
Ahh.  I knew it had to be something easy like that.  I created a 'win' as you describe and it works perfectly.  Thank you.

When I start the moonlight app on my android device I get a screen allowing me to choose from the list of games in my gamestream setup.  This could be a useful enhancement to this excellent project. :-)

Thank you for creating this, keep up the great work. :-)

**Comment by Athou** [comment]
https://github.com/dead/moonlight-openelec-rpi2 seems to be better than what I didas it has better integration with kodi.

**Comment by Bite-A-Rock** [comment]
Wow, thank you.  I will give it a try tonight. :-)
